### ISSUE TITLE
Default values of the histogram bins in analysis

### ISSUE THREAD

**Issue by snilsn** [tobac-project/tobac#366]
I just clicked on the [API reference](https://tobac.readthedocs.io/en/latest/tobac.html#) in the documentation and noticed that the default bin edge values for all the histogram functions are these really long, hand-coded lists of numbers. This is not really a high priority, but it just doesn't look professional and could be done much more compactly with `np.arange()`.

I don't know if these functions are actually used that much, and maybe I'm the only one who's annoyed by this, but it's one of the first things you see when you look at the API.

**Comment by w-k-jones** [comment]
I agree, this also sticks out to me when checking the API reference! Best practice would be to make the default value `None` and then create the array inside the function, rather than have a mutable default keyword value
